core/src/main/resources/hudson/model/Computer/index.jelly

```diff
@@ -52,16 +53,6 @@ THE SOFTWARE.
         </j:choose>
       </div>
 
-      <h1>
-        <span class="icon-lg">
-          <l:icon src="${it.iconClassName}" />
-        </span>
-        ${it.caption}
-        <j:if test="${!empty(it.node.nodeDescription)}">
-          <span style="font-size:smaller">(${it.node.nodeDescription})</span>
-        </j:if>
-      </h1>
-
```

Comment:
Causes [JENKINS-70394](https://issues.jenkins.io/browse/JENKINS-70394)

Reply:
I does not see any reason, why? Shall I fixed it, or work somebody on that?

Reply:
Indirectly; it now is to the right of a `div.warning` with `position: relative` whose area overlaps this button and doesn't allow interacting with the button.

No clue how to resolve other than move elements around which wouldn't be a good fix.

Reply:
heh it'll be related to https://github.com/jenkinsci/jenkins/pull/6511#pullrequestreview-1233350846 with the pre-existing UI issue. This sort of thing has happened before, there's a box around one of the elements I think which can block interacting with others.

> Shall I fixed it, or work somebody on that?

Would be great if you could

Reply:
We will see, I will try it today (or better tonight ;-) )

Reply:
> as save is wider than the form

It's unclear to me what the problem identified in the linked comment is, as the Save button being to the right of the description field is everywhere in Jenkins:


<img width="2925" alt="Screenshot 2023-01-10 at 17 53 08" src="https://user-images.githubusercontent.com/1831569/211613665-439c117c-99c7-4072-b2bf-fb86edfe4911.png">

Reply:
After few months pause in Jenkins core I need little more time to setup my dev instance again. But anyway , I tries to fix this buttons. It breaks the view in all my combination. Therefore I tries something "new". The app-bar as describe here: https://weekly.ci.jenkins.io/design-library/AppBar/

In has implemented app-bar and bottomButtonBar as well to see how it can looks. Therefore you see the button twice.
![image](https://user-images.githubusercontent.com/89339813/211664090-c5196bf6-2d02-413f-bd95-969357f6fb46.png)
It works perfect. The question is : will be this applicable solution? And when which one is preferred. I personally like the bottomButtonBar, because it can be placed close to reason. But I my look-and-feel is not the best one (I am working more on backends in my daily business)


This is how it looks on "small" screen
![image](https://user-images.githubusercontent.com/89339813/211665317-1e738967-0bb3-4db3-8649-8c8dd5e68dd5.png)


and when you scroll up, you newer lost focus to this buttons:
![image](https://user-images.githubusercontent.com/89339813/211665700-dec98696-300d-40c0-a5c9-c797a87e04ac.png)

Anyway, please get me feedback here which one (or no one) solution will be fine. Then I will commit my changes and create a PR.

ps: sorry for wasting here, but I will spare time preparing PR and discussing there if it is fine or not. I thing this will be faster way, because many of your core guys are mentioned here ;-)

Reply:
I think app-bar would make the most sense. I don't see why those buttons would need to be sticky

Reply:
@timja you right, I play now with the panel (try to add node monitors as table in the same view) and definitely is better app-bar. That means PR coming soon.

Reply:
see PR #7577